x-pack/legacy/plugins/monitoring/server/plugin.js

```diff
@@ -72,7 +72,11 @@ export class Plugin {
       reserved: {
         privilege: {
```

Comment:
Same thing here, does `api` fall under this object as well? What structure will it get to distinguish `all` and `read`?

Reply:
The `privilege` is a https://github.com/elastic/kibana/blob/7.5/x-pack/plugins/features/server/feature_kibana_privileges.ts#L10

Reply:
Oh, right. Since the privilege is either granted or not 🤦‍♂makes sense.